build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "1.6.0-beta4"
+  version = "1.7.0-beta4"
```

Comment:
1.6 hasn't been released yet, so this is getting a bit messy.  I'll release 1.6 and then this PR can go through as 1.7

Reply:
```suggestion
  version = "1.7.0-beta"
```